cips/cip-21.md

```diff
@@ -83,7 +84,42 @@ Test cases will need to ensure that a user may not forge a incorrect signer, nor
 
 ## Reference Implementation
 
-TBC
+The go implementation makes the following modifications to the `Blob` proto:
+
+```diff
+message BlobProto {
+  bytes namespace_id = 1;
+  bytes data = 2;
+  uint32 share_version = 3;
+  uint32 namespace_version = 4;
++ // Signer is sdk.AccAddress that paid for this blob. This field is optional
```

Comment:
```suggestion
+ // Signer is sdk.AccAddress that signed this blob. This field is optional
```

Reply:
> So even though the fee granter paid for the blob to be included on-chain, the MsgPayForBlobs submitter (celestia1rgj3z5rfydj3xulhm6glqr0zs7j8rr9rfh6ekw) is the authored blob "signer".

So in this case shouldn't it be `// Signer is sdk.AccAddress that signed this blob`?